src/Controller/Admin/BlogController.php

```diff
@@ -100,7 +101,10 @@ public function new(Request $request): Response
             // See https://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
-            if ($form->get('saveAndCreateNew')->isClicked()) {
```

Comment:
Could we get more details about why the original code was not correct? Thanks!

Reply:
`phpstan` complained about this line with:

```
 ------ ---------------------------------------------------- 
  Line   src/Controller/Admin/BlogController.php             
 ------ ---------------------------------------------------- 
  104    Call to an undefined method                         
         Symfony\Component\Form\FormInterface::isClicked().  
 ------ ---------------------------------------------------- 
```

Reply:
I guess because the IDE does not found the referenced method `isClicked()` into `FormInterface` (displayed as warning), though we know that's a `ClickableInterface` instance, so I don't see nothing wrong in the original code, it's just a type-hint issue. IMO should be reverted.

Reply:
@yceruto This is an actual `phpstan` error. Without this change build will fail if I revert it.

Reply:
@yceruto the thing is, you know it is a ClickableInterface because you know that this field was added as a button. But static analysis does not know it.
So when adding static analysis as part of the CI, it becomes necessary to make the code work with it here.

Reply:
Yep, I understand, but type-hinting it as is does not guarantee that the method exists though. If we want do it correctly we might need to check if it's an instance of `ClickableInterface`, then overkill IMHO.

I'm in favor of use `phpstan` locally, it's great and help me found a lot of issues, but to have it as part of the CI seems excessive to me if it's necessary to modify an obvious code.

Reply:
@phpeek you could do the following to skip this analysis: `$form['saveAndCreateNew']->isClickable()` and keep it simple.

Reply:
@yceruto Sorry for the delay - I was away for holidays.

You're right with `$form['saveAndCreateNew']->isClicked()` array notation, `phpstan` is no longer complaining. However, I don't like this notation, nor I'd encourage people to use it, since with `->get()`, You're getting a benefit of `OutOfBoundsException` being thrown in case of non existent key.